<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDocDialog</class>
 <widget class="QDialog" name="AddDocDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Doctor</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Doctor's Information</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>91</y>
     <width>251</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>70</height>
    </size>
   </property>
   <property name="title">
    <string>Name:</string>
   </property>
   <widget class="QLineEdit" name="NameE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>230</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>167</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>70</height>
    </size>
   </property>
   <property name="title">
    <string>Address:</string>
   </property>
   <widget class="QLineEdit" name="ADDE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>230</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>243</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>70</height>
    </size>
   </property>
   <property name="title">
    <string>Specialization:</string>
   </property>
   <widget class="QLineEdit" name="SpecE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>230</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>319</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>70</height>
    </size>
   </property>
   <property name="title">
    <string>Phone Number:</string>
   </property>
   <widget class="QLineEdit" name="PhnoE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>230</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>395</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>70</height>
    </size>
   </property>
   <property name="title">
    <string>Email:</string>
   </property>
   <widget class="QLineEdit" name="EmailE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>230</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
